grunt-tasks/svg-to-react.js

```diff
@@ -3,52 +3,140 @@
 
 module.exports = function( grunt ) {
   grunt.registerMultiTask( 'svg-to-react', 'Output a react component for SVGs', function() {
-    var component = '';
-    var componentExample = '';
-    var filesDest;
+		let components = '';
+		let componentsExample = '';
+		let filesDest;
+		const prepareIndividualIcon = require( '../sources/react/template-individual-icon' );
+		const prepareAllIcons = require( '../sources/react/template-all-icons' );
+		const prepareDevDocsExample = require( '../sources/react/template-devdocs-example' );
+		const iconsThatNeedOffset = [
+			'gridicons-add-outline',
+			'gridicons-add',
+			'gridicons-align-image-center',
+			'gridicons-align-image-left',
+			'gridicons-align-image-none',
+			'gridicons-align-image-right',
+			'gridicons-attachment',
+			'gridicons-bold',
+			'gridicons-bookmark-outline',
+			'gridicons-bookmark',
+			'gridicons-calendar',
+			'gridicons-cart',
+			'gridicons-create',
+			'gridicons-custom-post-type',
+			'gridicons-external',
+			'gridicons-folder',
+			'gridicons-heading',
+			'gridicons-help-outline',
+			'gridicons-help',
+			'gridicons-history',
+			'gridicons-info-outline',
+			'gridicons-info',
+			'gridicons-italic',
+			'gridicons-layout-blocks',
+			'gridicons-link-break',
+			'gridicons-link',
+			'gridicons-list-checkmark',
+			'gridicons-list-ordered',
+			'gridicons-list-unordered',
+			'gridicons-menus',
+			'gridicons-minus',
+			'gridicons-my-sites',
+			'gridicons-notice-outline',
+			'gridicons-notice',
+			'gridicons-plus-small',
+			'gridicons-plus',
+			'gridicons-popout',
+			'gridicons-posts',
+			'gridicons-scheduled',
+			'gridicons-share-ios',
+			'gridicons-star-outline',
+			'gridicons-star',
+			'gridicons-stats',
+			'gridicons-status',
+			'gridicons-thumbs-up',
+			'gridicons-textcolor',
+			'gridicons-time',
+			'gridicons-trophy',
+			'gridicons-user-circle',
+			'gridicons-reader-follow',
+			'gridicons-reader-following'
+		];
+		const iconsThatNeedOffsetY = [
+			'gridicons-align-center',
+			'gridicons-align-justify',
+			'gridicons-align-left',
+			'gridicons-align-right',
+			'gridicons-arrow-left',
+			'gridicons-arrow-right',
+			'gridicons-house',
+			'gridicons-indent-left',
+			'gridicons-indent-right',
+			'gridicons-minus-small',
+			'gridicons-print',
+			'gridicons-sign-out',
+			'gridicons-stats-alt',
+			'gridicons-trash',
+			'gridicons-underline',
+			'gridicons-video-camera'
+		];
+		const iconsThatNeedOffsetX = [
+			'gridicons-arrow-down',
+			'gridicons-arrow-up',
+			'gridicons-comment',
+			'gridicons-clear-formatting',
+			'gridicons-flag',
+			'gridicons-menu',
+			'gridicons-reader',
+			'gridicons-strikethrough'
+		];
```

Comment:
Can we move the arrays to a dedicated file that we import here?

Reply:
Done at bed88b4